### DISCUSSION TITLE
Visual editor

### DISCUSSION THREAD

**Discussion by Mrgove10** [community/community#25572] [Actions]
<p>Hello,</p>
<p>I can’t seem to find the visual editor for github actions, I tried with the .workflow and .yml files and I just do not see the visual editor button or tab.</p>
<p>Thanks in advan ce !</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/mrgove10">@mrgove10</a>,</p>
<p>Thanks for being here! The visual editor was deprecated in v2, so there is no visual editor in GitHub Actions V2. We are making some investments in the workflow space - we recommend keeping an eye out on our changelog for future updates! <a href="https://github.blog/changelog/" rel="nofollow noopener">https://github.blog/changelog/</a>.</p>

**Comment by Mrgove10** [comment]
<p>Hello <a class="mention" href="/andreagriffiths11">@andreagriffiths11</a> </p>
<p>Thanks for the response, is there chances that it will be added back in the future ? It looks awesome in pictures and marketing material but would be even better if we could use it !</p>
<p>Thanks in advance !</p>
